Phoenix/PHSpace.m

```diff
@@ -223,8 +244,23 @@ - (void)removeWindows:(NSArray<PHWindow *> *)windows {
 }
 
 - (void)moveWindows:(NSArray<PHWindow *> *)windows {
-    CGSMoveWindowsToManagedSpace(
-        CGSMainConnectionID(), (__bridge CFArrayRef)[self identifiersForWindows:windows], self.identifier);
+    CGSConnectionID cid = CGSMainConnectionID();
+    if ([NSProcessInfo canMoveWindowsToManagedSpace]) {
+        CGSMoveWindowsToManagedSpace(cid, (__bridge CFArrayRef)[self identifiersForWindows:windows], self.identifier);
+        return;
+    }
+
```

Comment:
Can you split the workaround to a separate private method to increase readability?

Reply:
Done!